### PR TITLE
Don't throw an exception when the project could not be found

### PR DESCRIPTION
**Pull request by diba1013**
instead just return an empty list

### CODE REVIEW

File: src/main/java/de/retest/recheck/configuration/ProjectConfiguration.java [hunk 1]
```diff
@@ -40,10 +40,12 @@ public static ProjectConfiguration getInstance() {
 		return instance;
 	}
 
+	public Optional<Path> getProjectConfigFolder() {
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/configuration/ProjectConfiguration.java:43]: Find it hard to distinguish between `getProjectConfigFolder()` and `findProjectConfigFolder()`, only the return types differ.

As the latter potentially throws an exception (like `Optional#get()`) and the former doesn't, how about:

* `getProjectConfigFolder()` => `searchProjectConfigFolder()` returns `Optional`.
* `findProjectConfigFolder()` => `getProjectConfigFolder()` potentially throws an exception.

I'm not sure, ideas?

Comment (diba1013) [reply src/main/java/de/retest/recheck/configuration/ProjectConfiguration.java:43]: I would have removed it, or at least made it private. But I could not be bothered to look at where it is used...

Comment (diba1013) [reply src/main/java/de/retest/recheck/configuration/ProjectConfiguration.java:43]: The `get...` is related to `ProjectRootFinderUtil.getProjectRoot()` where also an `Optional` is returned.

Comment (beatngu13) [reply src/main/java/de/retest/recheck/configuration/ProjectConfiguration.java:43]: OK, I see, makes sense!

**Review by beatngu13 — APPROVED** [review]

**Pull request merged**